Header header
obstacleInfo[] objects
nav_msgs/OccupancyGrid occupancyGrid
